Datadog.Trace.proj

```diff
@@ -16,7 +16,7 @@
     <ExcludeLegacyRedisProject Remove="test/test-applications/regression/**/StackExchange.Redis.AssemblyConflict.LegacyProject.csproj" />
     <CsharpTestHelperProject Include="test\**\*.TestHelpers.csproj"/>
     <CsharpToolsProject Include="tools\Datadog.Core.Tools\Datadog.Core.Tools.csproj"/>
-    <CppProject Include="src\**\*.vcxproj"/>
+    <CppProject Include="src\Datadog.Trace.ClrProfiler.Native\*.vcxproj"/>
```

Comment:
Not required, this is only because the normal tracer build was trying to build the nativeloader as well and was causing issues, so this was to remove them from the tracer build. Now that the tracer moved to nuke build system I can revert this change.

Reply:
fixed in 480803d